Day1.ps1

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) 2021 Ace Olszowka
+# How many measurements are larger than the previous measurement?
+
+function Measure-LargerThanPrevious {
+    [CmdletBinding()]
+    param (
+        [int[]]$Inputs
+    )
+    process {
+        $measurementsLargerThanPrevious = 0
+
+        # We special case the first value to do nothing
+        Write-Verbose -Message "$($Inputs[0]) (N/A - no previous measurement)"
+
+        # Now enumerate though each one
+        for ($i = 1; $i -lt $Inputs.Count; $i++) {
+            if ($Inputs[$i] -gt $Inputs[$i - 1]) {
+                Write-Verbose -Message "$($Inputs[$i]) (increased)"
+                $measurementsLargerThanPrevious++
+            }
+            elseif ($Inputs[$i] -eq $Inputs[$i - 1]) {
+                # The specification didn't say what to do in this scenario? Be silent?
```

Comment:
From what I understand about AoC if the specs don't mention it, it's safe to assume the scenario won't happen.

Reply:
Sounds legit; I just realized that once you register and submit something you get "more" of the puzzle; I'm trying to figure out how I want to lay this out....

Maybe a Day1.Part1.ps1 vs Day1.ps1; in theory if I didn't squash I could have a creative Git History that shows this; what did you do?

Reply:
I didn't spend too much thought on that part, each puzzle I have has 1 file that has output for part 1 and 2